pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -145,6 +145,8 @@ type OperatorConfigurationData struct {
 	ResyncPeriod               Duration                     `json:"resync_period,omitempty"`
 	RepairPeriod               Duration                     `json:"repair_period,omitempty"`
 	Sidecars                   map[string]string            `json:"sidecar_docker_images,omitempty"`
+	EnableSidecars             bool                         `json:"enable_sidecars"`
```

Comment:
I think it should be a `*bool*` for smooth upgrade path, otherwise you won't be tell apart if field is not specifies vs field is set to false